anycloud/cli/src/common/mod.rs

```diff
@@ -2,22 +2,40 @@ use std::env;
 use std::fs::read;
 use std::process::Command;
 
-async fn get_file(file_path: &str) -> Result<Vec<u8>, String> {
-  let pwd = env::current_dir();
-  match pwd {
-    Ok(pwd) => match read(format!("{}/{}", pwd.display(), file_path)) {
-      Ok(file) => Ok(file),
-      Err(_) => Err(format!("No Dockerfile at {}", pwd.display()).into()),
-    },
-    Err(_) => {
-      warn!(InvalidPwd, "Current working directory value is invalid");
-      std::process::exit(1);
+use tempdir::TempDir;
+
+async fn get_file(file_name: &str, file_path: Option<&str>) -> Result<Vec<u8>, String> {
+  match file_path {
+    Some(file_path) => {
+      println!("{}", format!("{}/{}", file_path, file_name));
+      let output = Command::new("ls")
+        .arg(format!("{}/{}", file_path, file_name))
+        .output()
+        .unwrap();
+      println!("{:?}", output);
```

Comment:
any reason for these prints or for the `ls`?

Reply:
No reason at all, I used it for testing and forgot to remove it